src/main/resources/resourcepacks/retromc/assets/cardstock/lang/en_us.json

```diff
@@ -1,47 +1,86 @@
 {
   "card.cardstock.retromc.appliedenergistics": "Applied Energistics 1",
+  "text.retromc.appliedenergistics.1": "The inventor of digital storage,",
+  "text.retromc.appliedenergistics.2": "AppEng solved a problem endemic to",
+  "text.retromc.appliedenergistics.3": "many modpacks of the time and today.",
   "card.cardstock.retromc.biosphere": "Biosphere",
-  "card.cardstock.retromc.industrialcraft": "IndustrialCraft 1",
-  "card.cardstock.retromc.redpower": "RedPower 2",
-
-  "card.cardstock.retromc.beekeeping": "Beekeeping",
-  "card.cardstock.retromc.factorization": "Factorization",
-  "card.cardstock.retromc.magicalcrops": "Magical Crops",
-  "card.cardstock.retromc.mffs": "Modular Force Field Systems",
-
-  "card.cardstock.retromc.terrafirmacraft": "TerraFirmaCraft",
-  "card.cardstock.retromc.thaumcraft3": "Thaumcraft 3",
-  "card.cardstock.retromc.witchery": "Witchery",
-  "card.cardstock.retromc.yttr": "Yttr",
-
-  "card.cardstock.retromc.dartcraft": "DartCraft",
-  "card.cardstock.retromc.qcraft": "QCraft",
-  "card.cardstock.retromc.universalelectricity": "Universal Electricity",
-  "card.cardstock.retromc.witchinggadgets": "Witching Gadgets",
-
-  "card.cardstockretromc.artifice": "Artifice",
-  "card.cardstock.retromc.mariculture": "Mariculture",
-  "card.cardstock.retromc.metallurgy": "Metallurgy",
-  "card.cardstock.retromc.steampower": "FlaxBeard's Steam Power",
-
   "text.retromc.biosphere.1": "Risugami's most well-known mod,",
   "text.retromc.biosphere.2": "and one of the first world generators",
   "text.retromc.biosphere.3": "providing a total system overhaul.",
+  "card.cardstock.retromc.industrialcraft": "IndustrialCraft 1",
+  "text.retromc.industrialcraft.1": "The first true tech mod,",
+  "text.retromc.industrialcraft.2": "its once-hated voltage system",
+  "text.retromc.industrialcraft.3": "is now appreciated by many.",
+  "card.cardstock.retromc.redpower": "RedPower 2",
   "text.retromc.redpower.1": "The all-time most impactful redstone mod,",
   "text.retromc.redpower.2": "its successor is a full game",
   "text.retromc.redpower.3": "that is still under development today!",
 
+  "card.cardstock.retromc.beekeeping": "Beekeeping",
   "text.retromc.beekeeping.1": "The first Minecraft mod to add bees,",
   "text.retromc.beekeeping.2": "a feature so popular that they're now",
   "text.retromc.beekeeping.3": "part of vanilla Minecraft itself.",
+  "card.cardstock.retromc.factorization": "Factorization",
   "text.retromc.factorization.1": "The progenitor of item barrels,",
   "text.retromc.factorization.2": "Factorization is appreciated by many",
   "text.retromc.factorization.3": "as one of MC's most underrated mods.",
+  "card.cardstock.retromc.magicalcrops": "Magical Crops",
+  "text.retromc.magicalcrops.1": "Renewable resources allowed",
+  "text.retromc.magicalcrops.2": "many of our favorite skyblock",
```

Comment:
first person feels weird tbh

Reply:
Oh yeah, lemme change that quick! one second~